test/utils/shallowEquality.spec.js

```diff
@@ -0,0 +1,138 @@
+import expect from 'expect';
+import shallowEqualScalar from '../../src/utils/shallowEqualScalar';
+import shallowEqual from '../../src/utils/shallowEqual';
+
+describe('Utils', () => {
+  // More info: https://github.com/gaearon/redux/pull/75#issuecomment-111635748
+  describe('shallowEqualScalar', () => {
+    it('returns true if both arguments are the same object referentially', () => {
+      const o = {a: 1, b: 2};
+      expect(shallowEqualScalar(o, o)).toBe(true);
+    });
+
+    it('returns false if either argument is null', () => {
+      expect(shallowEqualScalar(null, {})).toBe(false);
+      expect(shallowEqualScalar({}, null)).toBe(false);
+    });
+
+    it('returns false if only one argument is not an object', () => {
```

Comment:
Do we need this test? The way we use them in Redux, we make a hard assumption that both arguments have `'object'` `typeof`.